src/collective/vdexvocabulary/tests.py

```diff
@@ -25,7 +22,6 @@ def test_suite():
                 filename,
                 optionflags=optionflags,
                 globs={
-                    "interact": interact,
```

Comment:
I do not know why this was removed :)

Reply:
interact is only to have a python shell in doctests. mostly useful for developing or debugging doctests. should not a necessary dependency. https://pypi.org/project/interlude/